src/gei/Commands/GenerateScriptCommand.cs

```diff
@@ -38,6 +40,24 @@ public GenerateScriptCommand(OctoLogger log, ISourceGithubApiFactory sourceGithu
                 IsRequired = true,
                 Description = "Uses GH_PAT env variable."
             };
+
+            // GHES migration path
+            var ghesApiUrl = new Option<string>("--ghes-api-url")
+            {
+                IsRequired = false,
+                Description = "If migrating from GHES, the api endpoint for the hostname of your GHES instance. For example: http(s)://api.myghes.com"
+            };
+            var azureStorageConnectionString = new Option<string>("--azure-storage-connection-string")
+            {
+                IsRequired = false,
+                Description = "(Required when used with --ghes-api-url) The connection string for the Azure storage account, used to upload data archives pre-migration. For example: DefaultEndpointsProtocol=https;AccountName=myaccount;AccountKey=mykey;EndpointSuffix=core.windows.net"
+            };
+            var noSslVerify = new Option("--no-ssl-verify")
+            {
+                IsRequired = false,
+                Description = "(Only effective when passed in with --ghes-api-url) Disables SSL verification. If your GHES instance has a self-signed SSL certificate then setting this flag will allow data to be extracted. All other migration steps will continue to verify SSL."
```

Comment:
I'd suggest making `Disables SSL verification` to be `Disables SSL verification when communicating with your GHES instance. All other migration steps will continue to verify SSL. If your GHES instance has a self-signed SSL certificate then setting this flag will allow data to be extracted.`.

Reply:
Updated in https://github.com/github/gh-gei/pull/248/commits/8b741a3a51d6d7a94f5af881c2a3872afa5a17fa